src/dom_types/values.rs

```diff
@@ -25,14 +25,47 @@ impl<T: ToString> From<T> for CSSValue {
     }
 }
 
-// `&` because `style!` macro automatically adds prefix `&` before values for more ergonomic API
-// (otherwise it would fail when you use for example a Model's property in View functions as `CSSValue`)
-impl From<&CSSValue> for CSSValue {
-    fn from(value: &CSSValue) -> Self {
-        value.clone()
+// ----------- ToCssValue impls ------------
+
+// impl ToCssValue for CSSValue
+#[doc(hidden)]
+pub trait ToCssValue {
```

Comment:
I think we should rename it to `ToCssValueForCssValue` if we want to use the same pattern for all these traits.

Reply:
Make sense, Should it be `ToCssValueForCSSValue` ? or `ToCssValueCssValue` ? I prefer the later one but the type name is `CSSValue`.

Reply:
The goal is to make naming consistent. So we should use `*CSSValue*` or `*CssValue*` everywhere. (E.g. `ToCSSValueforCSSValue` or `ToCssValueForCssValue`)

I think that using `*CSSValue*` everywhere make more sense because we don't make breaking changes in public API - am I right?

Reply:
Ok, I will rename it to `ToCSSValueForCSSValue` to stay consistent with `CSSValue`, and I will leave `ToCss*` for another PR in the future if desired.